docs/serving/configuration/feature-flags.md

```diff
@@ -65,7 +65,7 @@ GA stage
 An extension surfaces details of a specific Knative implementation, or features of the underlying environment.
 
 !!! note
-    Extensions are never included in the core Knative API due to their lack of portability.
+Extensions are never included in the core Knative API due to their lack of portability.
```

Comment:
the prior indent is intentional - otherwise the note block doesn't render properly